### ISSUE TITLE
Doesnt work well with Zipline

### ISSUE THREAD

**Issue by mahadevans87** [ranaroussi/yfinance#364]
I have a Conda environment with python 3.5 for zipline. When I install yfinance via pip, it replaces a version of pandas and numpy that zipline uses. Hence zipline stops working. Is there any workaround for this?

**Comment by felixsilberstein** [comment]
have you tried installing them in virtual environment?

**Comment by mahadevans87** [comment]
Yes. I ended up installing yfinance and zipline in different environments

**Comment by ValueRaider** [comment]
Does this problem still exist?

**Comment by ValueRaider** [comment]
zipline needs to specify its requirements, then PIP will avoid installing conflict packages.
